src/locales/en.json

```diff
@@ -77,11 +77,11 @@
     "distance_header": "Distance",
     "duration_header": "Duration",
     "duration_question": "How long is the activity, in minutes?",
-    "intimate_risk_warning": "We have applied a minimum Activity Risk for kissing due to the risk involved in exchanging fluids.",
```

Comment:
This is very unclear, and a bit of guesswork went into my suggestion.

Reply:
What is unclear?

Reply:
Minimum is easily confused for "minimal", and I misread the latter part to mean an escalation to sex.
The "We" doesn't have to be invoked.

Reply:
I think we could probably drop that warning.